src/components/SharedComponents/VideoPlayer/VideoPlayer.tsx

```diff
@@ -148,13 +147,15 @@ const VideoPlayer: FC<VideoPlayerProps> = (props) => {
   if (videoToken.loading || thumbnailToken.loading || storyboardToken.loading) {
     return (
       <OakFlex
-        $flexDirection={"column"}
-        $width={"100%"}
-        $height={["all-spacing-19"]}
         $alignItems={"center"}
         $justifyContent={"center"}
+        $ba={"border-solid-l"}
+        $borderColor={"black"}
+        style={{
+          aspectRatio: "16/9",
+          boxSizing: "content-box",
+        }}
```

Comment:
I intend to make these available as props of `OakBox` since they're really useful for sizing a box.